pyproject.toml

```
@@ -0,0 +1,60 @@
+[project]
+name = "unitycatalog-function"
+version = "0.0.1"
+description = "Official python library for unitycatalog functions support"
+authors = [
+    { name="Serena Ruan", email="serena.ruan@databricks.com" }
+]
+readme = "README.md"
+requires-python = ">=3.8"
```

```suggestion
requires-python = ">=3.9"
```

3.8 will reach EOL soon.

---

lol just found it's Oct 2024!